tests/ts_calibrate.c

```diff
@@ -160,6 +161,11 @@ static int ts_validate(struct tsdev *ts, int boundary, unsigned int loops, int t
 
 	if (loops == 0)
 		loops = VALIDATE_LOOPS_DEFAULT;
+	if ((edgebuf + 1) > ((int)xres/2) || (edgebuf + 1) > ((int)yres/2)) {
```

Comment:
I think that this should also do a lower bounds check in case EDGE_BUFFER_MIN changes in the future:
```
if (edgebuf < EDGE_BUFFER_MIN || (edgebuf + 1) > ((int)xres/2) || (edgebuf + 1) > ((int)yres/2)) {
```

Reply:
Added lower bounds check for `edgebuf`